vies/locale/nl/LC_MESSAGES/django.po

```diff
@@ -17,24 +17,31 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-#: __init__.py:111
+#: models.py:21
+msgid "A VIES VAT field."
+msgstr "Een VIES BTW-nummer veld."
+
+#: types.py:114
 #, python-format
 msgid "%s is not a valid ISO_3166-1 country code."
 msgstr "%s is geen geldige ISO_3166-1 landcode."
 
-#: __init__.py:114
+#: types.py:117
+#, python-format
+msgid "%s is not a european member state."
+msgstr "%s is geen europese lidstaat."
+
+#: types.py:129
 #, python-format
-msgid "%s is not a VIES member country."
-msgstr "%s is geen VIES-lidstaat."
+msgid "%s does not match the country's VAT ID specifications."
+msgstr "%s komt niet overeen qua landcode en BTW specificaties."
```

Comment:
`komt niet overeen qua landcode en BTW specificaties` -> `komt niet overeen met de in dit land geldende BTW specificaties`?